### ISSUE TITLE
Lightbox slidenav inaccessible on touchscreen

### ISSUE THREAD

**Issue by pdomzal** [uikit/uikit#1162]
I'm new to this but I love uikit! Implemented the lightbox with a group of several images. It works great on the desktop but on mobile (Safari on iOS) the next/previous arrows don't display (no hover, of course). Is there a way to make the next/previous arrows remain always displayed?

**Comment by Gomah** [comment]
I had the same problem, fixed with css:
```
@media (max-width: 480px) {
   .uk-slidenav-position .uk-slidenav {display:block}
}
```

**Comment by pdomzal** [comment]
Thank you! 

I tried that. I don't mind having the slidenav arrows always on, so I used the CSS without the @media query. It works on desktop (Chrome on Windows, Safari on OSX) but the arrows still don't show in iOS (8.3 on iPhone 6 and iPad 2). And having the arrows in iOS is really the reason for doing this.

PS: I also had to cover the arrows to hide them where they were now showing from under the thumbnails.
